charts/alertmanager/values.yaml

```
@@ -173,6 +173,15 @@ topologySpreadConstraints: []
 statefulSet:
   annotations: {}
 
+  ## Statefulset's persistent volume claim retention policy
+  ## pvcDeleteOnStsDelete and pvcDeleteOnStsScale determine whether
+  ## statefulset's PVCs are deleted (true) or retained (false) on scaling down
+  ## and deleting statefulset, respectively. Requires 1.27.0+.
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#persistentvolumeclaim-retention
+  ##
+  pvcDeleteOnStsDelete: false
+  pvcDeleteOnStsScale: false
```

can we have it like this please?
```suggestion
  persistentVolumeClaimRetentionPolicy:
    whenDeleted: Delete
    whenScaled: Delete
```